source/docs/v3/integrations/livewire.blade.md

```diff
@@ -29,3 +31,49 @@ Now you can use Livewire both in the central app and the tenant app.
 Also make sure to enable [universal routes]({{ $page->link('features/universal-routes') }}).
 
 And if you're using file uploads, read the [Real-time facades]({{ $page->link('realtime-facades') }}) page of the documentation. Livewire uses real-time facades in the uploading logic.
+
+## Tenancy by path
+
+Open the `config/livewire.php` file and change this:
+
+```php
+'middleware_group' => ['web'],
+```
+
+to this:
+
+```php
+'middleware_group' => [
+    'web',
+    InitializeTenancyByPath::class,
+],
+```
+
+(Don't forget to import the middleware class.)
+
+Open the `config/livewire.php` file and add this:
+
+```php
+use Livewire\Controllers\HttpConnectionHandler;
+
+Route::group([
+    'prefix' => '/{tenant}',
+    'middleware' => [
+        'web',
+        InitializeTenancyByPath::class,
+    ],
+], function () {
+    Route::post('livewire/message/{name}', [HttpConnectionHandler::class, '__invoke']);
+});
+```
+
+Finally, open the blade file where the `@@livewireScripts` directive is, and add:
+
+```html
+@@livewireScripts
+<script>
+    window.livewire_app_url = '{{ tenant()->id }}';
```

Comment:
This got me stuck while trying the integration guide. The URL had two prefixes – the tenant ID + the tenant ID which gets assigned to `window.livewire_app_url` here (so the URL was `http://testing.test/foo/foo/testing`, the second `foo` would change with `window.livewire_app_url`, so if the tenant was `foo`, and we set `window.livewire_app_url` to `bar`, the URL would be `http://testing.test/foo/bar/testing`).

This suggestion fixes the problem 

```suggestion
    window.livewire_app_url = '/{{ tenant()->id }}';
```